docroot/themes/custom/mass_admin_theme/js/conditional-fields.js

```diff
@@ -141,4 +141,29 @@
     }
   };
 
+  /**
+   * Enable/disable fields on info details.
+   *
+   * Hide or show the banner field (field_banner_image)
+   * based on the field_details_enable_fields value.
+   */
+  Drupal.behaviors.enableInfoFieldsConditional = {
+    attach: function (context) {
+      $('.field--name-field-details-enable-fields', context).change(function () {
+        if ($(this).find('input').prop('checked') === false) {
+          // $(this).siblings('.field--name-field-banner-image').hide().find('.fieldgroup').removeAttr('required');
+          $(this).siblings('.field--name-field-banner-image').hide();
+          $(".field--name-field-info-details-sections details#section-content .field--name-field-section-long-form-content input[id$='-subform-field-section-long-form-content-add-more-add-more-button-callout-link']").hide();
+          $(".field--name-field-info-details-sections details#section-content .field--name-field-section-long-form-content input[id$='-subform-field-section-long-form-content-add-more-add-more-button-info-details-card-group']").hide();
+        }
+        else {
+          // $(this).siblings('.field--name-field-tabl-alignment').show().find('.fieldgroup').attr('required', 'required');
```

Comment:
@arthurbaghdas Do we need this commented out code?

Reply:
No we do not need, I've just copied from top for reference and forgot to clean up my code. Thank you